src/random_test.cpp

```diff
@@ -455,6 +457,21 @@ std::string Column::rand_value() {
   return "";
 }
 
+/* prepare single quoted string for LIKE clause */
+std::string Table::prepare_string(int where, std::string str) {
```

Comment:
what is the use str?  you are not using it.

Reply:
Also, this would cause a copy to be be passed, better semantics would be `const std::string & str` or pass by constant reference.

Reply:
I have used call by reference instead of call by const reference, since using const was not allowing me to modify the string in the function.